### DISCUSSION TITLE
SignIn to Citroen-Account not possible

### DISCUSSION THREAD

**Discussion by ich77** [flobz/psa_car_controller#997] [Q&A]
Hallo,
after Using psa_car_conector for a lot of month, I have to sign in again after a upgrade of my Raspberry Pi (Docker-compose). This didn't work. I have tried a new installation of docker-compose and a new download of the psa-car-controller but I allways get this error:
![grafik](https://github.com/user-attachments/assets/fad21b98-5b20-4e87-a0bc-a4159117af41)
sometimes also without any text in the red box.
Can somebody help?
Thanks
Markus

**Comment by Anistruk** [comment]
Same problem here, the addon had logout and now i can't login again.

**Comment by Haluska77** [comment]
My Citroen account is accessible, but returned
'No vehicle in your account is compatible with this API, you vehicle is probably too old...'

for Citroen Spacetourer 2019

**Comment by chmtc94** [comment]
May be look to this alternative  😉 GitHub - andreadegiovine/homeassistant-stellantis-vehicles: Integrate Stellantis vehicles on Home Assistant 
  
|  
|   
|   
|   |    |

   |

  |
|  
|    |  
GitHub - andreadegiovine&#47;homeassistant-stellantis-vehicles: Integrate Stellantis vehicles on Home Assistant
 
Integrate Stellantis vehicles on Home Assistant. Contribute to andreadegiovine&#47;homeassistant-stellantis-vehicles development by creating an account on GitHub.
  |   |

  |

  |

  


Envoyé depuis Yahoo Mail pour iPhone


Le mardi, mars 18, 2025, 9:31 AM, Stanislav Haluska ***@***.***> a écrit :




My Citroen account is accessible, but returned
'No vehicle in your account is compatible with this API, you vehicle is probably too old...'

for Citroen Spacetourer 2019
